tools/ci/download_bazel.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# Copyright 2019 The Numpy Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
```

Comment:
I think if you are the author of this file and wish to donate it to NumPy, you can skip the license preamble altogether since all of NumPy is covered by the license in the top-level directory. In any case, we cannot include code that is Apache-licensed (maybe this gets an exception since it is only CI? @rgommers, additional clarifications or corrections?

Reply:
You are probably right, CI only could get an exception, but frankly it seems not worth the trouble to go there for such a small file...

Reply:
But to be sure: This code is fully fresh written, or is it based on some blueprint out there? Because that would be a license violation.

Reply:
Yeah the code is all fresh although I took help from babel docs. So should I remove the license ?

Reply:
I would recommend you remove the license clause, you as author have the final say

Reply:
@mattip Removed the license template.